src/utils/FormatDate.ts

```diff
@@ -0,0 +1,11 @@
+import { month } from '@/constants/ProgressUpdates';
+
+export function FormatDate(): string {
```

Comment:
Function name and function work is not matching like function return the yesterday date and your write the function name is `FormtatDate()` it should be like `getYesterdayDate()` ?